pages/questionThumbnail/questionThumbnail.js

```diff
@@ -0,0 +1,16 @@
+var path = require('path');
+var express = require('express');
+var router = express.Router();
+
+router.get('/:qid/:filename', function(req, res, _next) {
+    var qid = req.params['qid'];
+    var filename = req.params['filename'];
+    var clientFilesDir = path.join(
+        res.locals.course.path,
+        'questions',
+        qid,
+    );
+    res.sendFile(filename, {maxAge: 86400000 * 30, root: clientFilesDir});
```

Comment:
I think this is a great simplification, but I'm worried this will serve any file in the same directory.  Could the route just get `/:qid` and then load the thumbnail path via sql query?  I think you can see an example of this pattern in `pages/news_items/news_items.js` and the sql file in same folder.